docs/file-conventions/-server.md

```diff
@@ -4,8 +4,9 @@ title: "*.server.ts extension"
 
 # `*.server.ts`
 
-While not always necessary, you can use `*.server.ts` on file names to force them out of client bundles. Usually the compiler is fine, but if you've got a server dependency with module side-effects, move it into a `your-name.server.ts` file to ensure it is removed from client bundles.
+While not always necessary, you can use `*.server.ts` on file names to force them out of client bundles. Usually the compiler is fine, but if you've got a server dependency with module side effects, move it into a `your-name.server.ts` file to ensure it is removed from client bundles.
 
-See [Route Module][routemodule] for more information.
+See [Route Module][route-module] for more information.
 
-[routemodule]: ../route/route-module
+<!-- page doesn't exist anymore -->
+[route-module]: ../route/route-module
```

Comment:
This page (https://remix.run/docs/en/dev/route/route-module) doesn't seem to exist anymore

Reply:
Same